### ISSUE TITLE
Coverage with nyc does not report any lines with uvu

### ISSUE THREAD

**Issue by pablo-abc** [atk/solid-register#5]
First of all, thanks for the work! I've been wanting to try `uvu` and this package gave me the perfect opportunity!

I'm currently trying to get code coverage for a package but if I use `nyc` uvu either fails with the `template is not a function` error (which does not happen if I don't run it with nyc). Although it did manage to show a bit of coverage. I managed to make it run with the following:

```json
    "test": "node --conditions browser node_modules/uvu/bin.js -r solid-register tests -i common",
    "test:ci": "nyc pnpm test"
```

But now `nyc` reports 0 lines covered:

```
-----------|---------|----------|---------|---------|-------------------
File       | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-----------|---------|----------|---------|---------|-------------------
All files  |       0 |        0 |       0 |       0 |
 index.tsx |       0 |        0 |       0 |       0 |
-----------|---------|----------|---------|---------|-------------------
```

---

My related deps:

```json
    "nyc": "^15.1.0",
    "solid-js": "^1.2.0",
    "solid-register": "^0.1.5",
    "uvu": "^0.5.3"
```

And I also have the following config for this package:

```json
  "solid-register": {
    "aliases": {
      "solid": "browser"
    }
  }
```

**Comment by atk** [comment]
This is unfortunately a known issue in the DOM expression compiler that the source maps for the JSX do not work, which is caused by a shortcoming of the babel engine (every compiled expression has to be connected to a source expression, but the template strings can often be related to multiple source expressions).

You could for testing purposes switch to solids h-implementation and resolve JSX in the client and get coverage, but then the tested code differs from the end product significantly.

Long story short, this is not caused by and cannot be solved by solid-register. I'd love to have coverage, too.
